api/proto/lightnode/lightnode.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package lightnode;
+import "common/common.proto";
+option go_package = "github.com/Layr-Labs/eigenda/api/grpc/lightnode";
+
+///////////////////////////////////////////////////////////////////////////////////
+// Warning: light node APIs are currently in flux. This is not yet a stable API. //
+///////////////////////////////////////////////////////////////////////////////////
+
+service LightNode {
+	// GetChunk retrieves a specific chunk held by the light node.
+	rpc GetChunk(GetChunkRequest) returns (GetChunkReply) {}
+
+	// StreamAvailabilityStatus streams the availability status of all chunks assigned to the light node.
+	// For use by a DA node for monitoring the availability of chunks through its constellation of agent light nodes.
+	rpc StreamAvailabilityStatus(StreamAvailabilityStatusRequest) returns (stream StreamAvailabilityStatusReply) {}
+}
+
+// A request for a specific chunk from a light node.
+message GetChunkRequest {
+	bytes header_hash = 1;
+}
+
+// A reply to a GetChunk request.
+message GetChunkReply {
+	// The chunk data.
+	common.ChunkData chunk = 1;
+}
+
+// A request from a DA node to an agent light node to stream the availability status of all chunks
```

Comment:
Why is this API specific to agent light node? Should it applicable to all light nodes?

Reply:
Good to know this, it may be worth documenting the difference of behavior between agent vs. non-agent LNs.